boards.txt

```diff
@@ -20649,3 +20649,210 @@ esp32c3m1IKit.menu.EraseFlash.all=Enabled
 esp32c3m1IKit.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
+
+Edgebox-ESP-100.name=Edgebox-ESP-100
+
+Edgebox-ESP-100.bootloader.tool=esptool_py
+Edgebox-ESP-100.bootloader.tool.default=esptool_py
+
+Edgebox-ESP-100.upload.tool=esptool_py
+Edgebox-ESP-100.upload.tool.default=esptool_py
+Edgebox-ESP-100.upload.tool.network=esp_ota
+
+Edgebox-ESP-100.upload.maximum_size=1310720
+Edgebox-ESP-100.upload.maximum_data_size=327680
+Edgebox-ESP-100.upload.flags=
+Edgebox-ESP-100.upload.extra_flags=
+Edgebox-ESP-100.upload.use_1200bps_touch=false
+Edgebox-ESP-100.upload.wait_for_upload_port=false
+
+Edgebox-ESP-100.serial.disableDTR=false
+Edgebox-ESP-100.serial.disableRTS=false
+
+Edgebox-ESP-100.build.tarch=xtensa
+Edgebox-ESP-100.build.bootloader_addr=0x0
+Edgebox-ESP-100.build.target=esp32s3
+Edgebox-ESP-100.build.mcu=esp32s3
+Edgebox-ESP-100.build.core=esp32
+Edgebox-ESP-100.build.variant=Edgebox-ESP-100
+Edgebox-ESP-100.build.board=Edgebox-ESP-100
+
+Edgebox-ESP-100.build.usb_mode=0
```

Comment:
Change `Edgebox-ESP-100.build.usb_mode=0` to 1 to match changes in next comment.

Reply:
The latest commit should have the changes as request, Thanks.